universal-login-example/src/Daefen.js

```diff
@@ -0,0 +1,123 @@
+/*
```

Comment:
Why not add as npm?

Reply:
wasn’t sure of the proper way of adding this... It’s not an npm library yet

Reply:
wasn’t sure of the proper way of adding this... It’s not an npm library yet

Reply:
I that case let's move it to `src/vendors` and add that dire to linter ignore